melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/buc/BucUtils.java

```diff
@@ -31,4 +40,27 @@
             !document.getConversations().isEmpty()
             && document.getConversations().get(0).getVersionId() != null
             && lovvalgSedTyper.contains(document.getType());
+
+    private static final Pattern VERSJON_PATTERN = Pattern.compile("^v(\\d)\\.(\\d)$");
```

Comment:
Her må mønsteret endres når vi når versjoner med flersifrede tall, f.eks. v10.1 eller v4.10. Kan evt. ha `"^v(\\d+)\\.(\\d+)$"`. Sikkert ikke stor sjanse for at de endrer til så høyt versjonstall i nær fremtid.

Reply:
Hehe stemmer det. Jeg tipper dog at vi har et tiår eller mer på oss der 😄 Jeg tenkte bare å ha det så konkret som mulig da vi kun forventer ett siffer for GVer og Ver